python/runtime/maxcompute.py

```diff
@@ -32,8 +43,7 @@ def read_feature(raw_val, feature_spec):
                                         sep=feature_spec["delimiter"])
                 indices = indices.reshape(indices.size, 1)
                 values = np.ones([indices.size], dtype=np.int32)
-                dense_shape = np.array(feature_metas[name]["shape"],
-                                       dtype=np.int64)
+                dense_shape = np.array(feature_spec["shape"], dtype=np.int64)
```

Comment:
Seems that we can remove the `read_feature` method?

Reply:
Will do in the next PR.

Reply:
`read_feature ` is called by `read_features_from_row` which will be used when predicting to get columns that are used as features to feed to the model.